app/process/model/transaction/transaction.go

```diff
@@ -18,22 +18,24 @@ package transaction
 
 type (
 	HederaTransaction struct {
-		ConsensusTimestamp   string `json:"consensus_timestamp"`
-		TransactionHash      string `json:"transaction_hash"`
-		ValidStartTimestamp  string `json:"valid_start_timestamp"`
-		ChargedTxFee         int    `json:"charged_tx_fee"`
-		MemoBase64           string `json:"memo_base64"`
-		Result               string `json:"result"`
-		Name                 string `json:"name"`
-		MaxFee               string `json:"max_fee"`
-		ValidDurationSeconds string `json:"valid_duration_seconds"`
-		Node                 string `json:"node"`
-		TransactionID        string `json:"transaction_id"`
-		Transfers            []Transfer
+		ConsensusTimestamp   string     `json:"consensus_timestamp"`
+		TransactionHash      string     `json:"transaction_hash"`
+		ValidStartTimestamp  string     `json:"valid_start_timestamp"`
+		ChargedTxFee         int        `json:"charged_tx_fee"`
+		MemoBase64           string     `json:"memo_base64"`
+		Result               string     `json:"result"`
+		Name                 string     `json:"name"`
+		MaxFee               string     `json:"max_fee"`
+		ValidDurationSeconds string     `json:"valid_duration_seconds"`
+		Node                 string     `json:"node"`
+		TransactionID        string     `json:"transaction_id"`
+		Transfers            []Transfer `json:"transfers"`
+		TokenTransfers       []Transfer `json:"token_transfers"`
 	}
 	Transfer struct {
 		Account string `json:"account"`
 		Amount  int64  `json:"amount"`
+		Asset   string `json:"token_id"`
```

Comment:
Maybe we should add comment here that for `Transfers` this property is not populated